README.md

```diff
@@ -13,35 +13,51 @@ var response = {
   items: [
     {
       someValue: 'wow',
-      someLink: {sys: {type: 'Link', linkType: 'Entry', id: 'suchId'}}
+      someLink: { sys: { type: 'Link', linkType: 'Entry', id: 'suchId' } }
     }
   ],
   includes: {
     Entry: [
-      {sys: {type: 'Entry', id: 'suchId'}, very: 'doge'}
+      { sys: { type: 'Entry', id: 'suchId' }, very: 'doge' }
     ]
   }
 };
 
 var items = resolveResponse(response)
+// Responds with the original object in the first position and mutated object in the second.
```

Comment:
What's the intention to change the response interface of the exposed method?

Reply:
was thinking on the lines of if there is a need for the original copy of the object that went in. ? Remember changing it on basis of a specific use case i had in mind, but unable to recall it now. Let me know your thoughts on this.

Reply:
As soon as this function call doesn't mutate the object, you basically have the variable `response` holding the original object you passed in, so I don't see any benefit in having the exact same thing returned.
The `resolved` key from the returned object is the thing you want to get back.

```js
const response = { ... }
const items = resolveResponse(response)

deepEqual(reponse, items.response) // true
```

See, you already have the object around, there's no need to get it returned.
The following is way more concise:

```js
const response = { ... }
const resolved = resolveResponse(response)
```

Reply:
I agree with @zcei